app/models/proposal.rb

```diff
@@ -206,6 +206,14 @@ def partial_approve
     end
   end
 
+  # Is the user getting emails for this proposal
+  def is_user_tracking? user
+    observers = self.observations.map {|o| o.user}
+    users = self.users - self.delegates + observers
+    users.include? user
```

Comment:
Note that `self.observers` obviates the need for line 211. Given the math in 212, seems like spelling out exactly who you want might make more sense than trying to remove one group and add a potentially-intersecting group back in.

There's a nuance here; comments are only sent to the requester, observers, and approvers _who have been notified already_ (see `Comment#listeners`). For example, if a tier 2 approver added a comment, they wouldn't be "subscribed" until after tier 1 approved.